packages/core/src/actions.ts

```diff
@@ -292,7 +294,7 @@ export function sendTo<
   TEvent extends EventObject,
   TActor extends AnyActorRef
 >(
-  actor: string | TActor | ((ctx: TContext) => TActor),
+  actor: string | TActor | ((ctx: TContext, event: TEvent) => TActor),
```

Comment:
Should we add a test case that would check that `event` param can be used to get the `TActor`? I'm pretty sure that it just works but it would be good to have such test to validate the types